### PR TITLE
Added basic driving code

### PR DESCRIPTION
**Pull request by billwpierce**
Added basic driving code. Ports need to be changed to what we actually use this year.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 1]
```diff
@@ -1,30 +1,70 @@
...
 
-		public static class Pneumatics {}
+		public static class Pneumatics {
+			public static final int shifterUp = 4;
+			public static final int shifterDown = 5;
+		}
 	}
 
-	public static class Metrics {}
+	public static class Metrics {
```
Comment (oldgalileo) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: Please don't include unknown metrics.

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 2]
```diff
@@ -1,30 +1,70 @@
...
 		}
 
 		public static class CANMotor {}
 
-		public static class PWM {}
+		public static class PWM {
+			public static final int leftDriveA = 1;
+			public static final int leftDriveB = 2;
+			public static final int rightDriveA = 3;
+			public static final int rightDriveB = 4;
```
Comment (oldgalileo) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: Set these to default values for right now (`-1`)

**Review by oldgalileo — CHANGES_REQUESTED** [review]

File: src/org/usfirst/frc4904/robot/RobotMap.java (hunk 1 above)
Comment (ajnadel) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: These are the real values.

File: src/org/usfirst/frc4904/robot/RobotMap.java (hunk 2 above)
Comment (billwpierce) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: fixed

**Review by oldgalileo — APPROVED** [review]
Looks good.

**Pull request merged**